dev/TeachingTip/TeachingTip.cpp

```diff
@@ -748,25 +749,35 @@ void TeachingTip::UpdateDynamicHeroContentPlacementToTop()
 {
     if (HeroContentPlacement() == winrt::TeachingTipHeroContentPlacementMode::Auto)
     {
-        winrt::VisualStateManager::GoToState(*this, L"HeroContentTop"sv, false);
-        if (m_currentHeroContentEffectivePlacementMode != winrt::TeachingTipHeroContentPlacementMode::Top)
-        {
-            m_currentHeroContentEffectivePlacementMode = winrt::TeachingTipHeroContentPlacementMode::Top;
-            TeachingTipTestHooks::NotifyEffectiveHeroContentPlacementChanged(*this);
-        }
+        DoUpdateDynamicHeroContentPlacementToTop();
+    }
+}
+
+void TeachingTip::DoUpdateDynamicHeroContentPlacementToTop()
```

Comment:
We normally call these types of method *Impl()

Reply:
Done.